dev/fattest.simplicity/bnd.bnd

```diff
@@ -84,4 +88,13 @@ test.project: true
 	org.jboss.arquillian.core:arquillian-core-spi;version=1.3.0.Final,\
 	org.jboss.arquillian.core:arquillian-core-api;version=1.3.0.Final,\
     org.jboss.arquillian.test:arquillian-test-spi;version=1.3.0.Final,\
-    org.testcontainers:testcontainers;version=1.12.3
+    org.testcontainers:testcontainers;version=1.12.3,\
+    org.testcontainers:database-commons;version=1.12.3,\
+	org.testcontainers:jdbc;version=1.12.3,\
+	org.testcontainers:testcontainers;version=1.12.3,\
+	org.testcontainers:db2;version=1.12.3,\
+	org.testcontainers:postgresql;version=1.12.3,\
+	org.testcontainers:mssqlserver;version=1.12.3,\
+	org.testcontainers:oracle-xe;version=1.12.3,\
+	org.rnorth.duct-tape:duct-tape;version=1.0.7,\
+	org.slf4j:slf4j-api;version=1.7.7
```

Comment:
I know these 2 dependencies are needed at runtime, but are they needed at compile-time here on the -buildpath?